### DISCUSSION TITLE
can you pass values from _app to component getServerSideProps??

### DISCUSSION THREAD

**Discussion by willybeans** [vercel/next.js#14135] [Help]
I am currently passing values from _app to page component using getInitialProps via the getInitialProps function in _app (psuedocode):
```
class myApp extends App {
static async getInitialProps({ Component, context }) {
let passingValues = await Component.getInitialProps({ ...context, myPassedValues })
return { passingValues };
}
render()
return()
}
```

via the Component prop received by the function in line 2, i can pass myPassedValues to the component being rendered in line 3. Does this same functionality exist for getServerSideProps? Is it possible to have _app have visibility to the getServerSideProps function in the component to pass values or does that not exist yet? I know that _app currently cannot perform 'getServerSideProps', but that does strictly prohibit it from interacting with/sending values to the other pages that perform it?

**Comment by AronBe** [comment]
I would be also interested if this is possible, @willybeans have you found the answer?
